Proofreading MO Guide (#11605)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md

# Converting a TensorFlow XLNet Model {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_XLNet_From_Tensorflow}

Pretrained models for XLNet (Bidirectional Encoder Representations from Transformers) are
[publicly available](https://github.com/zihangdai/xlnet).

## Supported Models

> The following models from the [pretrained XLNet model list](https://github.com/zihangdai/xlnet#pre-trained-models) are currently supported:

Reviewer: ```suggestion
The following models from the pretrained [XLNet model list](https://github.com/zihangdai/xlnet#pre-trained-models) are currently supported:
```

2. Save and run the following Python script in *`~/XLNet-Base/xlnet`*:

> > **Note**: The original model repository has been tested with TensorFlow 1.13.1 under Python2.

Reviewer: ```suggestion
> **NOTE**: The original model repository has been tested with TensorFlow 1.13.1 under Python2.
```

```

## Downloading the Pretrained Large XLNet Model

Download and unzip an archive with the [XLNet-Large, Cased](https://storage.googleapis.com/xlnet/released_models/cased_L-24_H-1024_A-16.zip).

After unzipping the archive, the directory *`cased_L-12_H-1024_A-16`* is created and contains the following files:

* TensorFlow checkpoint (*`xlnet_model.ckpt`*) containing the pretrained weights (which is actually 3 files)
* sentence piece model (*`spiece.model`*) used for (de)tokenization
* config file (*`xlnet_config.json`*) which specifies the hyperparameters of the model

> To get pb-file from the archive contents, you need to do the following.

Reviewer: ```suggestion
To get *`pb-file`* from the archive contents, follow the below instructions:
```

Author: ```suggestion
To get *`pb-file`* from the archive contents, follow the instructions below:
```
The word "below" should succeed the noun.
